recipe/meta.yaml

```diff
@@ -11,7 +11,7 @@ source:
   sha256: 0cb4478d33090cdd2aad138e6f8b672a289cc76fd6c6313a63155f60c12ae6af
```

Comment:
We also need to update the `sha256` here.

Reply:
How?

Reply:
It can be found here:

![image](https://user-images.githubusercontent.com/591645/133299329-5ea02fb4-c490-41ec-9345-93fada5afb04.png)

https://pypi.org/project/jupyterlab-js-logs/#copy-hash-modal-14d263ec-21ec-4d7d-a90a-5396e05234cb

Reply:
ooh, thanks!!